lib/viewsBundler.js

```diff
@@ -3,6 +3,7 @@ require('@colors/colors')
 // Views Bundler: Module to expose templates that are available on the server down to the client as well
 
 module.exports = app => {
+  const fs = require('fs')
```

Comment:
i believe `fse` defined on the next line allows you to do all the same stuff as `fs`, so this line is irrelevant. all the `fs` calls below can be done from `fse` with an identical api